nss-day-cohort-46/rare-rest-rare-api-cool-cake
# feat: user can create categories. Categories are listed alphabetically.

# Description

User can create a new category.
Categories are listed alphabetically.

Closes #6 

## Type of change


# Testing Instructions

`git fetch --all`
`git checkout cr-categories-create`
`pytthon3 manage.py loaddata categories`
`python3 manage.py runserver`

For client-side instructions, see frontend pull request.

In Postman, make a `GET` request to view all current categories at

`
http://localhost:8000/categories
`
Then make a `POST` request to create a new category. 
```json
{
  "label": "New Label"
}
```

Make another `GET` request to verify category was created.

# Checklist:
